src/coverlet.MTP/coverlet.MTP.csproj

```diff
@@ -0,0 +1,102 @@
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn)</NoWarn>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+
+    <SuppressDependenciesWhenPacking>false</SuppressDependenciesWhenPacking>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);CopyProjectReferencesToPackage</TargetsForTfmSpecificContentInPackage>
+
+    <EnablePackageValidation>false</EnablePackageValidation>
+    <!--<IsTestProject>true</IsTestProject>-->
+    <TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>
+    <IsPackable>true</IsPackable>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <!-- disable transitive version update and use versions defined in coverlet.core -->
+    <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
+    <!-- settings for Muse.coverlet.MTP.globalconfig -->
+    <!-- <CodeAnalysisTreatWarningsAsErrors>false</CodeAnalysisTreatWarningsAsErrors>
+    <AnalysisMode>Recommended</AnalysisMode>
+    <AnalysisLevel>latest-major</AnalysisLevel>
+    <Deterministic>true</Deterministic>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources> -->
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>coverlet.MTP</Title>
+    <PackageId>coverlet.MTP</PackageId>
+    <Authors>tonerdo</Authors>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/coverlet-coverage/coverlet</PackageProjectUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/tonerdo/coverlet/master/_assets/coverlet-icon.svg?sanitize=true</PackageIconUrl>
+    <PackageIcon>coverlet-icon.png</PackageIcon>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <Description>coverage code coverage for Microsoft Testing Platform</Description>
+    <PackageTags>coverage;microsoft-testing-platform;code-coverage</PackageTags>
+    <PackageReadmeFile>Coverlet.MTP.Integration.md</PackageReadmeFile>
+    <PackageReleaseNotes>https://github.com/coverlet-coverage/coverlet/blob/master/Documentation/Changelog.md</PackageReleaseNotes>
+    <RepositoryType>git</RepositoryType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)..\..\_assets\coverlet-icon.png" Pack="true" PackagePath="\" />
+    <None Include="..\..\Documentation\Coverlet.MTP.Integration.md" Link="Coverlet.MTP.Integration.md" Pack="true" PackagePath="\">
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
+    <PackageReference Include="Microsoft.Testing.Platform" />
```

Comment:
This might cause tooling to think that coverlet.MTP is a test project.

To avoid that, set `IsTestingPlatformApplication` property to false.